README.md

```diff
@@ -359,10 +358,30 @@ To release new version run
 grunt uglify bump
 
 ```
-
 ### Testing
 
-Open `test/SpecRunner.html` in your web browser to run Jasmine test suite.
+
```

Comment:
Today I'll focus on the README. Yesterday I was patching it. Today I'll read it top to bottom and do it properly.